tests/chainermn_tests/communicator_tests/test_communicator.py

```diff
@@ -155,6 +176,42 @@ def __init__(self, param):
         'batched_copy': True,
     }]]
 
+
+gpu_mixed_dtype_params = [Param(p) for p in [
+    {
```

Comment:
`PureNcclCommunicator` is not in `gpu_mixed_dtype_params ` . Does `PureNcclCommunicator`  need to be tested?

Reply:
Yes, and `PureNcclCommunicator` cases are appended after line 202.
This is because only `PureNcclCommunicator` supports FP16 communication natively, so more exhaustive tests are necessary than for other communicators.

Reply:
OK!